guides/prefix-cache-storage/cpu/README.md

```diff
@@ -129,6 +129,8 @@ The following benchmark results demonstrate the performance improvements of usin
 *   **vLLM Configuration:**
     *   `gpu_memory_utilization` was set to `0.65`.
     *   CPU offloading was enabled with `num_cpu_blocks` set to `41000`, which provides approximately 100GB of CPU cache.
+*  ** LMCache configuration: **
+    *   For LMCache setup, `LMCACHE_MAX_LOCAL_CPU_SIZE` is set to 100 GB.
 
```

Comment:
Are these both using the same blocksize?  (ISTR lmcache defaults to 64, whereas vLLM default is 16?).  

The results are surprisingly different for an otherwise comparable setup - I'd expect some, but such a large difference seems like it may not be apples-to-apples comparison, for some reason.